build_keyboard.mk

```diff
@@ -198,10 +198,14 @@ endif
 
 ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
     OPT_DEFS += -DRGBLIGHT_ENABLE
-    SRC += $(QUANTUM_DIR)/light_ws2812.c
     SRC += $(QUANTUM_DIR)/rgblight.c
     CIE1931_CURVE = yes
     LED_BREATHING_TABLE = yes
+ifeq ($(strip $(RGBLIGHT_CUSTOM_DRIVER)), yes)
+			OPT_DEFS += -DRGBLIGHT_CUSTOM_DRIVER
+else
+			SRC += $(QUANTUM_DIR)/light_ws2812.c
+endif
```

Comment:
There's a mix of spaces and tabs here, the rest of the file uses spaces, so could you please chang it to spaces?